README.md

```diff
@@ -277,3 +277,85 @@ importing and reuse.
 |               03 | 2010-04-30     |
 |               02 | 2009-12-07     |
 |               01 | 2009-10-28     |
+
+## ChEMBL Downloader Status
```

Comment:
duplicate table?

Reply:
I though you wanted the table in the README? Or should I just remove it and just print it when the script runs?